src/components/ContestTile.js

```diff
@@ -55,27 +54,30 @@ const ContestTile = ({ contest: { node }, updateContestStatus }) => {
           />
         ) : (
           <p>
-            Contest ran {t.startDay}-{t.endDay}
+            Contest ran {t.startDay === "" ? "" : t.startDay}-
+            {t.endDay === "" ? "" : t.endDay}
           </p>
         )}
         <ClientOnly>
           <Link
-            to={contestPath}
+            to={fields?.contestPath || "/"}
             className="contest-repo button button-small cta-button primary"
           >
             {`${findingsRepo === "" ? "Preview" : "View"} Contest`}
           </Link>
           {t.contestStatus === "active" && contestRepo && (
-            <a
-              href={contestRepo}
+            <Link
+              to={contestRepo}
               className="button button-small cta-button secondary"
             >
               View Repo
-            </a>
+            </Link>
           )}
-          {t.contestStatus === "active" && findingsRepo && submissionPath ? (
+          {(t.contestStatus === "active" || status === "Active Contest") &&
+          findingsRepo &&
+          fields.submissionPath ? (
             <Link
-              to={submissionPath}
+              to={fields?.submissionPath || "/"}
```

Comment:
another: when submissionPath is expected, it should exist.

Reply:
Right, It cause an error for some reason.

So I think I can keep it this way knowing that that's part of the team's checklist to have this info today !
![Capture d’écran 2022-07-12 à 06 12 49](https://user-images.githubusercontent.com/59468684/178407006-1eadc80a-8874-4143-8a8d-b5aa9ea364e1.png)